data/autoyast_sle12/autoyast_wicked.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <!--  very minimal autoyast profile-->
+  <networking>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+  </networking>
+  <bootloader>
+    <device_map config:type="list">
+      <device_map_entry>
+        <firmware>hd0</firmware>
+        <linux>/dev/sda</linux>
+      </device_map_entry>
+    </device_map>
+    <global>
+      <timeout config:type="integer">1</timeout>
+      <append> splash=verbose showopts</append>
+    </global>
+  </bootloader>
+  <scripts>
+  <chroot-scripts config:type="list">
+    <script>
+        <chrooted config:type="boolean">true</chrooted>
+        <interpreter>shell</interpreter>
+        <filename>modify_grub.sh</filename>
+        <feedback config:type="boolean">true</feedback>
+        <feedback_type>error</feedback_type>
+        <source>
+          <![CDATA[
+            #!/bin/sh
+            sed -i "s#GRUB_CMDLINE_LINUX_DEFAULT=\"[^\"]*\"#GRUB_CMDLINE_LINUX_DEFAULT=\"video=1024x768-16 plymouth.ignore-serial-consoles console=ttyS0 console=tty splash=verbose showopts\"#" /etc/default/grub
+            sed -i "s/# GRUB_DISABLE_LINUX_UUID=true/GRUB_DISABLE_LINUX_UUID=true/" /etc/default/grub
+            grub2-mkconfig -o /boot/grub2/grub.cfg
+          ]]>
+        </source>
+       </script>
+  </chroot-scripts>
+  </scripts>
+  <partitioning config:type="list">
+    <drive>
+      <device>/dev/sda</device>
+      <disklabel>msdos</disklabel>
+      <initialize config:type="boolean">true</initialize>
+      <partitions config:type="list">
+        <partition>
+          <mountby config:type="symbol">device</mountby>
+          <filesystem config:type="symbol">swap</filesystem>
+          <mount>swap</mount>
+        </partition>
+        <partition>
+          <mountby config:type="symbol">device</mountby>
+          <filesystem config:type="symbol">ext4</filesystem>

```

Comment:
Still not sure if it is good idea to replace btrfs by ext4. Maybe could you try to disable btrfsmaintenance services in autoyast?

Reply:
why ? there is tons of tests testing btrfs . this test not FS related.

Reply:
actually I find it wrong that 99% of our test cases are running on single FS type . No matter that it is our "lovely" BTRFS . Still we stating that we also support others but only very specific FS tests running on different FS's majority using single one...